zds/tutorialv2/views/views_published.py

```diff
@@ -125,6 +125,7 @@ def get_context_data(self, **kwargs):
                                           if reaction.author == self.request.user]
 
         context['isantispam'] = self.object.antispam()
+        context['pm_link'] = self.object.get_absolute_contact_url(u'À propos de')
```

Comment:
Super.

Du coup tu peux juste encadrer ta chaîne de caractères dans la fonction 
d'internationalisation stp? (`_(u'À propos de')`)
Le 04/04/2016 01:31, Anto59290 a écrit :
>
> In zds/tutorialv2/views/views_published.py 
> <https://github.com/zestedesavoir/zds-site/pull/3486#discussion_r58319723>:
>
> > @@ -125,6 +125,7 @@ def get_context_data(self, **kwargs):
> >                                            if reaction.author == self.request.user]
> >
> >          context['isantispam'] = self.object.antispam()
> > +        context['pm_link'] = self.object.get_absolute_contact_url(u'À propos de')
>
> Ca surcharge le titre par defaut du MP qui est "Collaboration" - Titre 
> du contenu, ici ca devient "A propos de" - Titre du contenu
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3486/files/548127f45f6b04900d620d87a9a0a17e4de8a402#r58319723>
>